src/modelAnalysis/rumba/rumba.m

```diff
@@ -0,0 +1,294 @@
+function [RUMBA_outputs, UpRegulated, DownRegulated, MetConnectivity1, MetConnectivity2] = rumba(model1, model2, completeModel, sampling, maxMetConn, RxnsOfInterest, GenesOfInterest, NormalizePointsOption, PValCuttoff, MaxNumPoints, LoopRxnsToIgnore, verboseTag)
+% RUMBA predicts which reactions significantly change their

```

Comment:
Could you use a inputParser here, I can easily imagine someone wanting to set the verbose output for a default input, at which point it would be necessary to provide all 8 other optional arguments with their correct default values just to get a verbose output. If you want help, just let me know.